config/notify.yml

```diff
@@ -0,0 +1,10 @@
+<% fake_api_key = "fake_api_key-00000000-0000-0000-0000-000000000000-00000000-0000-0000-0000-000000000000" %>
```

Comment:
Does notify complain if the key is not in this format? just curious if we can get away with something simpler.

Reply:
Yes unfortunately Notify complains about an invalid API key if it's not in a format that's something along the lines of "account_name-long-uuid".